Inverted-Index/src/tests/Tester.java

```diff
@@ -0,0 +1,127 @@
+package tests;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.Test;
+import exceptions.NotADirectoryException;
+import file_handler.DirectoryReader;
+import file_handler.Doc;
+import file_handler.FileReader;
+import file_handler.IDirectoryReader;
+import search.InvertedIndex;
+import search.Mapper;
+import search.QueryParser;
+import search.SearchEngine;
+
+import org.junit.Assert;
+import org.junit.Before;
+
+public class Tester {
+    private List<Doc> testDocs;
+    private File[] testFiles;
+    @Before
+    public void getTestDocs() throws NotADirectoryException, FileNotFoundException {
+        IDirectoryReader reader = new DirectoryReader();
+        testFiles = reader.getFiles("C:\\Repos\\Summer1400-SE-Team24\\Inverted-Index\\src\\tests\\docs");
```

Comment:
the docs' path better to be constant and relative (to be able to run on other devices)